src/sql/src/session/vars/definitions.rs

```diff
@@ -1653,6 +1653,19 @@ pub mod cluster_scheduling {
         "Enables SecurityContext for clusterd instances, restricting capabilities to improve security.",
         false,
     );
+
+    const DEFAULT_CLUSTER_REFRESH_MV_COMPACTION_ESTIMATE: Duration = Duration::from_secs(1200);
```

Comment:
To err on the side of caution, we could default this to something small, like 60 seconds, and increase it only for folks with very large MatViews?
```suggestion
    const DEFAULT_CLUSTER_REFRESH_MV_COMPACTION_ESTIMATE: Duration = Duration::from_secs(60);
```

Reply:
Ok, I've changed it to 60 seconds.